### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by enoqueJonas**
# Math Magicians: tests

## This PR was raised to:
- Add unit tests for Math Magicians 

## What does this PR do:
- Write unit tests with React Testing Library.
- Write unit tests with snapshots with Jest.

## Improvements to be made:
- improve styling of the app

### CODE REVIEW

File: src/__test__/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+describe('Calculator component test', () => {
```
Comment (mwanawabangona) [inline src/__test__/Calculator.test.js:5]: - [ ] `required`
- [ ] Good job so far!! 🥂 
- [ ] However, I'm failing to run the test or start the project
- [ ] see the screenshot below and the highlighted error

`error screenshot`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @enoqueJonas,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by enoqueJonas** [comment]
Hello.React scripts was not working with the normal configuration in my terminal, so I made some changes to the scripts in package.jsonKindly change the start and test scripts to the normal configuration, I am sure it will work after that.On 16/12/2022, at 00:59, mwanawabangona ***@***.***> wrote:﻿
@mwanawabangona requested changes on this pull request.

Hi @enoqueJonas,
Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!
Highlights

✅  No linter errors ✔️
✅  Repo looks professional 💼
✅  Good quality commits

Required Changes ♻️
Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In src/__test__/Calculator.test.js:
> @@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from ***@***.***/react';
+import Calculator from '../components/Calculator';
+
+describe('Calculator component test', () => {


 required
 Good job so far!! 🥂
 However, I'm failing to run the test or start the project
 see the screenshot below and the highlighted error

error screenshot

**Comment by enoqueJonas** [comment]
Hello @mwanawabangona. React scripts was not working with the normal configuration in my terminal, so I made some changes to the scripts in package.json. Kindly change the **start** and **test** scripts to the normal configuration, I am sure it will work after that. On 16/12/2022, at 00:59, mwanawabangona ***@***.***> wrote:﻿
[@mwanawabangona](https://github.com/mwanawabangona) requested changes on this pull request.

**Pull request merged**